src/main/js/home.js

```diff
@@ -28,20 +28,11 @@
  * OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-/*globals $:false, document:false, window:false */
+document.addEventListener('DOMContentReady', function () {
```

Comment:
Codacy has a fix for the issue: [Replace `'DOMContentReady'` with `"DOMContentReady"`](https://app.codacy.com/gh/yegor256/rultor/pullRequest?prid=9694263)
```suggestion
document.addEventListener("DOMContentReady", function () {
```